### ISSUE TITLE
Greek letters

### ISSUE THREAD

**Issue by andrewadare** [jheinen/GR.jl#11]
The docs indicate that syntax like `\alpha` should render a Greek symbol (without requiring LaTeX). But I see the following:
```
import GR
GR.plot(collect(1:10), collect(1:10))
GR.textext(0.2, 0.8, "\\alpha, \\beta, \\gamma")
```
<img width="498" alt="example" src="https://cloud.githubusercontent.com/assets/154163/10832579/e9d563d0-7e51-11e5-92fb-b6fb01dbd9c2.png">

Is this a font issue? 
```
julia> versioninfo()
Julia Version 0.4.0
Commit 0ff703b* (2015-10-08 06:20 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin13.4.0)
  CPU: Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```

**Comment by jheinen** [comment]
Yes - you're right. The problem only occurs in GKSTerm - we'll fix it ASAP ...

**Comment by jheinen** [comment]
The bug has been fixed in e44b382.
